database/bugs.sql

```diff
@@ -0,0 +1,14 @@
+CREATE TABLE IF NOT EXISTS bugs (
```

Comment:
generate this from the go code eventually.  Use struct tags and reflection so that things don't accidentally get out of sync.

I'll show you how.

Reply:
Just noting that we talked offline and this has been added as a future task.